easybuild/easyconfigs/c/cwltool/cwltool-3.1.20221018083734-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,108 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'cwltool'
+version = '3.1.20221018083734'
+
+homepage = 'https://github.com/common-workflow-language/cwltool'
+description = """Common workflow language (CWL) reference implementation."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake',    '3.20.1'),
+    ('make',     '4.3'),
+    ('Check',    '0.15.2'),
+    ('SWIG',     '4.0.2'),
+    ('expat',    '2.2.9'),
+    ('bzip2',    '1.0.8'),
+    ('zlib',     '1.2.11'),
+]
+
+# The Qt5 dependency of Graphviz (required by pydot) may be long to build and
+# requires a lot of disk space and memory!
+dependencies = [
+    ('Python',            '3.9.5'),
+    ('typing-extensions', '3.10.0.0'),
+    ('pydot',             '1.4.2'),
+    ('lxml',              '4.6.3'),
+    ('ruamel.yaml',       '0.17.21'),
+    ('networkx',          '2.6.3', '', ('foss', '2021a')),
```

Comment:
remove the `('foss', '2021a')` part here after you change the `toolchain` above

Reply:
Done